src/data/analysis.json

```diff
@@ -65,5 +65,11 @@
       "openjdk17"
     ],
     "story_points": ""
+  },
+  "weblogic_to_eap7_export_csv": {
+    "path": "eap7/weblogic/tests/data",
+    "targets": ["eap7"],
+    "story_points": "",
+    "export_csv": true
```

Comment:
Hi Nandini, The object references was designed to read an options block, so if the json is updated this way:
```
   "options" : [
         "export_csv": true
]
```
then you won't need to add an if statement at line 121, and any advanced options will filled automatically when parsing the json,
what needs to be done is to check the options tag, and fill the default values; target, input, cli ... etc

Reply:
@midays , the analysis.json file is now updated .